persistence/src/main/java/edu/unc/lib/dl/persist/services/delete/MarkForDeletionJob.java

```diff
@@ -58,29 +63,30 @@ public MarkForDeletionJob(PID pid, AgentPrincipals agent,
 
     @Override
     public void run() {
-        aclService.assertHasAccess("Insufficient privileges to delete object " + pid.getUUID(),
-                pid, agent.getPrincipals(), markForDeletion);
+        try (Timer.Context context = timer.time()) {
```

Comment:
what, is there some shortage of indents that i'm unaware of? ;-) it's a fair suggestion, though i may leave this alone for now, since the run() method isn't too unwieldy as is